app/src/main/java/com/example/trashminder/utils/AlarmReceiver.kt

```diff
@@ -10,19 +10,25 @@ import android.os.Build
 import android.util.Log
 import androidx.core.app.NotificationCompat
 import com.example.trashminder.R
+import com.google.firebase.firestore.FieldValue
+import com.google.firebase.firestore.FirebaseFirestore
+import java.text.SimpleDateFormat
 import java.util.*
 import java.util.concurrent.TimeUnit
 
-
 class AlarmReceiver : BroadcastReceiver() {
 
+    private val firestore = FirebaseFirestore.getInstance()
+
     override fun onReceive(context: Context, intent: Intent?) {
```

Comment:
This `onReceive` function is getting quite long. Could you split it up? Rule of thumb is if you have to scroll your screen to see the whole functions, it's already too long ;)